docs/src/config/solver.md

```diff
@@ -138,21 +138,9 @@ number of eigenmodes of the problem. The available options are:
 
   - `"SLEPc"`
   - `"ARPACK"`
-  - `"FEAST"`
```

Comment:
RIP Feast.

Reply:
It can come back when needed! Commit c435ace0 has the original code which just needs minor modifications after the linear algebra refactors of last year.